src/components/modals/AdaptiveCardViewer/AdaptiveCardViewer.tsx

```diff
@@ -9,23 +9,21 @@ import { connect } from 'react-redux';
 import { Modal } from 'office-ui-fabric-react/lib/Modal';
 import { Template, RenderedActionArgument } from '@conversationlearner/models'
 import { State } from '../../../types'
-import * as AdaptiveCards from 'adaptivecards';
+import * as AdaptiveCards from 'adaptivecards'
 import { injectIntl, InjectedIntlProps } from 'react-intl'
-var hostconfig = require('./AdaptiveCardHostConfig.json')
-   
+import adaptiveCardHostConfig from './AdaptiveCardHostConfig'
+
 class AdaptiveCardViewer extends React.Component<Props, {}> {
     onDismiss = () => {
         this.props.onDismiss()
     }
 
-    renderOptions(): any {
-        return {
-            hostConfig: hostconfig
-        }
+    getAdaptiveCardHostConfig(): any {
+        return new AdaptiveCards.HostConfig(adaptiveCardHostConfig)
     }
 
     getTemplate(): any {
-        let templateString = JSON.stringify(this.props.template.body);
```

Comment:
This was the only questionable part. This looks like a mistake that the old renderer just happened to accept.  The template.body is already a string, so calling JSON.stringify on it escaped it further which mean the subsequent JSON.parse would only turn it back into a string instead of actual JSON which resulted in render failure.